Changelog.md

```diff
@@ -1,7 +1,7 @@
 ### 0.8.3 (unreleased)
 
 Language Features:
-
+ * Possibility to use ``bytes.concat`` with variable number of ``bytes`` and ``FixedBytes`` arguments which behaves as a restricted version of `abi.encodePacked` with a more descriptive name.
```

Comment:
Needs to be moved.

Reply:
Also `FixedBytes` is not a "keyword" and thus should not be used in typewriter font. You can use `bytesNN` or "fixed-length bytes".

Reply:
used `bytesNN` and moved